docs/cookbooks/common_workflows.md

```diff
@@ -9,28 +9,28 @@ For any workflow, the bigquery-etl repositiory needs to be locally available, fo
 The [Creating derived datasets tutorial](https://mozilla.github.io/bigquery-etl/cookbooks/creating_a_derived_dataset/) provides a more detailed guide on creating scheduled queries.
 
 1. Run `./bqetl query create <dataset>.<table>_<version>`
-    * Specify the desired destination dataset and table name for `<dataset>.<table>_<version>`
-    * Directories and files are generated automatically
-1. Open `query.sql` file that has been created in `sql/moz-fx-data-shared-prod/<dataset>/<table>_<version>/` to write the query
-1. [Optional] Run `./bqetl query schema update <dataset>.<table>_<version>` to generate the `schema.yaml` file
+   1. Specify the desired destination dataset and table name for `<dataset>.<table>_<version>`
```

Comment:
the numbers you can leave as `1`, MD will render them correctly. It just means in the future if we need to add an extra point in the middle then we don't have to update all the numbers.

Reply:
Agree, and I actually did. Pycharm keeps changing it, I'll check the Editor configuration.